e2e/jest.test-setup.ts

```diff
@@ -83,7 +83,7 @@ beforeEach(async () => {
 
   const notRedirectRequest = (request: Request): boolean => {
     // Response body can only be accessed for non-redirect requests
-    return request && request.redirectChain().length === 0;
+    return request && request.redirectChain().length === 0 && !request.isNavigationRequest();
```

Comment:
Fixing exception from reading console message when page has changed.
```
console.error
      Exception occurred when getting console messages.
      Error: Execution context was destroyed, most likely because of a navigation.

      153 |       // tslint:disable-next-line:no-empty
      154 |     } catch (err) {
    > 155 |       console.error(`Exception occurred when getting console messages.\n${err}`);
          |               ^
      156 |     }
      157 |   });
```